### PR TITLE
Update block_actions.js

### PR DESCRIPTION
**Pull request by SpikeThatMike**

### CODE REVIEW

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
can you run npm run fix ?

File: lib/plugins/block_actions.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ function inject (bot, { version }) {
   bot._client.on('block_action', (packet) => {
     const pt = new Vec3(packet.location.x, packet.location.y, packet.location.z)
     const block = bot.blockAt(pt)
+    if(block === null) return;
```
Comment (wvffle) [inline lib/plugins/block_actions.js:56]: Add newlines around the if. That way it'd be more readable.

Comment (wvffle) [reply lib/plugins/block_actions.js:56]: I'd also add `{` and `}`

**Review by wvffle — CHANGES_REQUESTED** [review]

**Comment by rom1504** [comment]
can you run npm run fix then commit the result?

**Review by rom1504 — APPROVED** [review]

**Pull request merged**